### PR TITLE
chore(precisebank): Add queries to swagger

### PR DESCRIPTION
**Pull request by nddeluca**
This adds the precisebank protobuf generated swagger documentation to the swagger combine configuration in order to be rendered in the swagger.yaml file.

### CODE REVIEW

**Review by drklee3 — APPROVED** [review]

File: client/docs/swagger-ui/swagger.yaml [hunk 1]
```diff
@@ -12019,6 +12019,163 @@ paths:
...
+        '200':
+          description: A successful response.
+          schema:
+            type: object
+            properties:
+              remainder:
+                description: >-
+                  remainder is the amount backed by the reserve, but not yet
+                  owned by any
+
```
Comment (sesheffield) [inline client/docs/swagger-ui/swagger.yaml:12093]: extra newline?

File: client/docs/swagger-ui/swagger.yaml [hunk 2]
```diff
@@ -12019,6 +12019,163 @@ paths:
...
+        '200':
+          description: A successful response.
+          schema:
+            type: object
+            properties:
+              total:
+                description: >-
+                  total is the total sum of all fractional balances managed by
+                  the precisebank
+
```
Comment (sesheffield) [inline client/docs/swagger-ui/swagger.yaml:12144]: extra newline?

File: client/docs/swagger-ui/swagger.yaml [hunk 3]
```diff
@@ -62082,6 +62239,56 @@ definitions:
...
+      QueryFractionalBalanceResponse defines the response type for
+      Query/FractionalBalance method.
+  kava.precisebank.v1.QueryRemainderResponse:
+    type: object
+    properties:
+      remainder:
+        description: >-
+          remainder is the amount backed by the reserve, but not yet owned by
+          any
+
```
Comment (sesheffield) [inline client/docs/swagger-ui/swagger.yaml:62263]: extra newline?

File: client/docs/swagger-ui/swagger.yaml [hunk 4]
```diff
@@ -62082,6 +62239,56 @@ definitions:
...
+      QueryRemainderResponse defines the response type for Query/Remainder
+      method.
+  kava.precisebank.v1.QueryTotalFractionalBalancesResponse:
+    type: object
+    properties:
+      total:
+        description: >-
+          total is the total sum of all fractional balances managed by the
+          precisebank
+
```
Comment (sesheffield) [inline client/docs/swagger-ui/swagger.yaml:62281]: extra newline?

File: client/docs/swagger-ui/swagger.yaml [hunk 5]
```diff
@@ -12019,6 +12019,163 @@ paths:
                       format: byte
       tags:
         - Liquid
+  /kava/precisebank/v1/fractional_balance/{address}:
+    get:
+      summary: >-
+        FractionalBalance returns only the fractional balance of an address.
+        This
+
```
[the remark this answers is not in the document]
Comment (drklee3) [reply client/docs/swagger-ui/swagger.yaml:12027]: They're from the proto comments spanning multiple lines - seems to render okay on swagger

Not sure if there's a config option for doc generation to remove the newlines

[the remark this answers is not in the document]
Comment (sesheffield) [reply client/docs/swagger-ui/swagger.yaml:12027]: that's fine I didn't realize they were auto generated

**Review by sesheffield — APPROVED** [review]

**Pull request merged**